pegasus/sites.v3/code.org/layouts/default.haml

```diff
@@ -33,15 +33,16 @@
     -if @header['theme'] == 'responsive' || 'responsive_full_width'
       -if @header['nav']
         %div
-          .col-80-right
+          %main.col-80-right
```

Comment:
This wraps the main content on pages with sub-navigation, like https://code.org/about.